src/main.py

```diff
@@ -0,0 +1,18 @@
+import psycopg2
+from flask import Flask,request,render_template
+from flask_restful import Resource,Api
+from user import *
+from community import *
+
+app = Flask(__name__)
+api = Api(app)
+
+api.add_resource(ShowComm,'/community/') # GET request to get community info.
```

Comment:
I'm not sure if Flask does this by default, but perhaps we should prefix these endpoints with `api/`

Reply:
I think we should prefix them by the API version `/v1/communities/` and then add api somewhere in the base url like `api.github.com`